app/controllers/partnering_organizations_controller.rb

```diff
@@ -52,6 +51,11 @@ def create
   # PATCH/PUT /partnering_organizations/1.json
   def update
     respond_to do |format|
+      PartnerCategory.where(partnering_organization_id: @partnering_organization.id).delete_all
+      params[:category_ids].each do |i|
+        @po_cat = PartnerCategory.new(partnering_organization_id: @partnering_organization.id, category_id: i)
+        @po_cat.save
```

Comment:
Also not sure if theres a way to do these saves at once? @lbkchen

Reply:
Don’t think so :/ u can use create instead of new and save though